src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -203,9 +209,23 @@ private boolean validateReportId(ConcurStandardAccountingExtractDetailLine line,
         }
         return valid;
     }
-
+    
     private boolean validateAccountingLine(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
         if (!getConcurStandardAccountingExtractCashAdvanceService().isCashAdvanceLine(line)) {
+            ConcurAccountInfo accountingInformation = buildConcurAccountingInformation(line, line.getJournalAccountCode(), line.getSubObjectCode());
```

Comment:
The way in which the sub-object code is being set up by buildConcurAccountingInformation() may need to be adjusted to allow for using the report/header sub-object code in the case of personal expense lines. I think I accidentally omitted handling for that when I modified this class to handle report/header accounting fields; sorry about that.